third_party/OWNERS

```diff
@@ -5,6 +5,7 @@ reviewers:
   - smarterclayton
   - thockin
 approvers:
+  - sig-architecture-approvers
```

Comment:
this is probably closer to dep-approvers

Reply:
good point